src/routes/identity/account/+page.svelte

```diff
@@ -63,6 +85,8 @@
 	<Container align="center" direction="row" gap={6} justify="center" padX={24}>
 		Assets <Badge dark>{$balanceStore.balances.length}</Badge>
 	</Container>
+	<Button on:click={getBalances}>Fetch balances</Button>
```

Comment:
Can't we fetch them `onMount` in the right page? Or on the whole page load. I don't like this here.